File: QE-SELENIUM-PYTHON/Activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Chrome()
driver.get("https://training-support.net/webelements/dynamic-controls")

print("Page Title:", driver.title)

checkbox = driver.find_element(By.ID, "checkbox")
print("Checkbox selected:", checkbox.is_selected())

checkbox.click()
print("Checkbox selected after click:", checkbox.is_selected())

driver.quit()

File: QE-SELENIUM-JAVA/demo/src/main/java/demos/Activity6.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity6 {
    public static void main(String[] args) {

        WebDriver driver = new ChromeDriver();
        driver.get("https://training-support.net/webelements/dynamic-controls");

        System.out.println("Page Title: " + driver.getTitle());

        WebElement checkbox = driver.findElement(By.id("checkbox"));
        System.out.println("Checkbox selected: " + checkbox.isSelected());

        checkbox.click();
        System.out.println("Checkbox selected after click: " + checkbox.isSelected());

        driver.quit();
    }
}